ITKImageProcessingFilters/ITKStitchMontage.h

```diff
@@ -61,7 +60,8 @@ class ITKImageProcessing_EXPORT ITKStitchMontage : public AbstractFilter
 {
   Q_OBJECT
   PYB11_CREATE_BINDINGS(ITKStitchMontage SUPERCLASS AbstractFilter)
-  PYB11_PROPERTY(IntVec3Type MontageSize READ getMontageSize WRITE setMontageSize)
+  PYB11_PROPERTY(IntVec2Type MontageStart READ getMontageStart WRITE setMontageStart)
```

Comment:
Adding in the Montage Start/End values we no longer need to select any DataContainers because those can now be programmatically generated. This lessens the chance that the user messes up the import. Right now the user must select the start/end values *AND* select the proper data containers to be used. Please update this filter so that the user does not need to select any Data Containers.